cmd/zoekt-git-index/main.go

```diff
@@ -107,6 +110,7 @@ func run() int {
 		opts.LanguageMap[m[0]] = ctags.StringToParser(m[1])
 	}
 
+	profiler.InitLightweight("zoekt-git-index", zoekt.Version)
```

Comment:
What's the reasoning for having a separate profiler config? Is mutex profiling polluting zoekt-git-index and that's not the case for webserver or indexserver?

Reply:
Agreed, interesting in why we need a lighterweight indexing config here. Given this will only trigger for slower runs of zoekt-git-index, more info seems useful?

Reply:
The thing I found concerning was `AllocForceGC: true`. If I'm understanding correctly, it means that enabling profiling could significantly impact memory usage (lower peak usage due to more frequent GCs). So you could have weird things like "we switched off the GCP profiler and now memory usage is much higher."

However, it looks like we do this all over the place anyways, and I haven't heard of a problem. I'm probably just scarred from my Java days where WAY more stuff is on the heap and GC behavior is very core to memory usage.

I'll remove `InitLightweight`!